Resources/Prototypes/SoundCollections/desceration.yml

```diff
@@ -0,0 +1,6 @@
+- type: soundCollection
+  id: desceration
```

Comment:
The word is desecration. 9 typos of this :P

Reply:
blame SS13